kyuubi-hive-jdbc/src/main/java/org/apache/kyuubi/jdbc/hive/Utils.java

```diff
@@ -42,7 +42,7 @@ public class Utils {
       Arrays.asList("jdbc:hive2://", "jdbc:kyuubi://");
 
   /** If host is provided, without a port. */
-  static final String DEFAULT_PORT = "10000";
+  static final String DEFAULT_PORT = "10009";
```

Comment:
Default value of `kyuubi.frontend.thrift.binary.bind.port`

https://github.com/apache/kyuubi/blob/5cbbdc32dbc74b7ab2a1cd6d71a7d0af07945494/kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala#L512